tutorial1.md

```diff
@@ -0,0 +1,299 @@
+# Tutorial 1
+
+## Add the apt repo
+
+```shell
+curl -s https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add -
+cat <<EOF >/etc/apt/sources.list.d/kubernetes.list
+deb https://apt.kubernetes.io/ kubernetes-xenial main
```

Comment:
Might want to add an installation step to ensure apt-transport-https is installed if we're using https here.

Reply:
It was installed by default on Bionic server

Reply:
But I am gonna add a `tutorial 0` that covers prerequisites